packages/network/src/resend/resendStrategies.ts

```diff
@@ -346,7 +346,7 @@ export class ForeignResendStrategy implements Strategy {
                 const nodeId = storageNodeIds.shift()!
                 try {
                     // eslint-disable-next-line require-atomic-updates
-                    storageNode = await this.nodeToNode.connectToNode(nodeId, tracker)
+                    storageNode = await this.nodeToNode.connectToNode(nodeId, tracker, false)
```

Comment:
Ah the need for this is going be removed very soon by this PR: https://github.com/streamr-dev/network-monorepo/pull/25 (but good to have it in here for the moment ofc)

Reply:
ok